app/models/miq_report.rb

```diff
@@ -252,6 +252,24 @@ def group_description=(_group_description)
     # which does not exist in the MiqReport class
   end
 
+  def columns_for_sorting(columns)
+    columns || sortby || col_order
+  end
+
+  def validate_sorting_columns(columns)
+    validate_columns(columns_for_sorting(columns))
+  end
+
+  def validate_columns(sorting_columns)
+    sorting_columns.split(",").collect do |attr|
```

Comment:
do other reporting methods handle the `split` of the columns?
for some reason this feels wrong for me.

I thought `columns` and `sortby` were an array. or at least `Array(column)` friendly. so I expect an array (or string) to come in here.

Reply:
ok, I also[ moved split to method columns_for_sorting](https://github.com/ManageIQ/manageiq/pull/18532/commits/e79365382960642b23d487194208c43fa85929aa) and added Array() to validate_columns also